packages/core/src/geometry/GeometrySystem.js

```diff
@@ -588,21 +588,32 @@ export default class GeometrySystem extends System
     {
         const { gl } = this;
         const geometry = this._activeGeometry;
+        const uints_for_indices = gl.getExtension("OES_element_index_uint");
```

Comment:
We keep the extension references in the `ContextSystem` class. There is a function called `getExtensions()` We should add it there as this check only happens once on startup. Placing it here means we check it for each draw call!